run-custom-mao.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash -xe
+
+if [ -z "$CUSTOM_MAO_IMAGE" ]; then
+  # Custom MAO image not already provided
+
+  if [ -z "${1}" ]; then
```

Comment:
A more compact way of doing this would be to default to $1, e.g

```
CUSTOM_MAO_IMAGE=${CUSTOM_MAO_IMAGE:-$1}
if [ -z "$CUSTOM_MAO_IMAGE" ]; then
...
```